### PR TITLE
Adding IO suites for baremetal pipeline

### PR DESCRIPTION
**Pull request by tintumathew10**
# Description

Adding IO suites for baremetal piepline

http://10.70.46.85:8080/view/Baremetal/job/qe-reef-cephfs-baremetal/2/console

### CODE REVIEW

File: suites/reef/baremetal/io/rh/dmfg_clara_1admin_5node_2client.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/reef/baremetal/io/rh/dmfg_clara_1admin_5node_2client.yaml:1]: incorrect description

File: suites/reef/baremetal/io/rh/extensa_clara_multisite_1admin_4node_1client.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Test suite for for baremetal rgw-multisite deployment and testing multisite sync at scale.
```
Comment (vamahaja) [inline suites/reef/baremetal/io/rh/extensa_clara_multisite_1admin_4node_1client.yaml:1]: incorrect description

File: suites/reef/baremetal/io/rh/nvme_cali_1admin_6node_2client.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/reef/baremetal/io/rh/nvme_cali_1admin_6node_2client.yaml:1]: incorrect description

File: suites/reef/baremetal/io/rh/rados_extensa_1admin_5node_1client.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Use cluster-conf file: conf/reef/rados/stretch-mode-host-location-attrs.yaml
```
Comment (vamahaja) [inline suites/reef/baremetal/io/rh/rados_extensa_1admin_5node_1client.yaml:1]: incorrect description

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/reef/baremetal/io/rh/nvme_cali_1admin_6node_2client.yaml (hunk 3 above)
Comment (tintumathew10) [reply suites/reef/baremetal/io/rh/nvme_cali_1admin_6node_2client.yaml:1]: done

File: suites/reef/baremetal/io/rh/extensa_clara_multisite_1admin_4node_1client.yaml (hunk 2 above)
Comment (tintumathew10) [reply suites/reef/baremetal/io/rh/extensa_clara_multisite_1admin_4node_1client.yaml]: done

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Review by vamahaja — APPROVED** [review]

**Pull request merged**